src/redux/watchList/actionCreator.js

```diff
@@ -0,0 +1,19 @@
+export const FETCH_WATCHLIST_SUCCESS = 'FETCH_WATCHLIST_SUCCESS';
+export const POST_WATCHLIST_SUCCESS = 'POST_WATCHLIST_SUCCESS';
+
+export function fetchWatchList() {
+  return {
+    type: FETCH_WATCHLIST_SUCCESS,
+  };
+}
+
+export function postWatchList(watchListItem) {
+  return {
+    type: POST_WATCHLIST_SUCCESS,
+    payLoad: {
+      id: 100,
```

Comment:
pls use some heuristic to generate the id, at least a random number, otherwise it will be pretty hard to delete/edit the watchlists